COMPILING.md

```diff
@@ -0,0 +1,161 @@
+Jamulus can be compiled for Linux, Windows and macOS as follows. Alernatively, you may wish to use one of the contributed [installation scripts](https://github.com/jamulussoftware/installscripts).
+
+---
+
+
+## Download sources
+
+For .tar.gz [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.tar.gz) to download the latest release
+
+For .zip [use this link](https://github.com/jamulussoftware/jamulus/archive/master.zip)
+
+## Linux
+
+#### Install dependencies
+
+On Ubuntu-based distributions 18.04+, Debian 9+ or 10 and Raspberry Pi Raspbian Buster release or later:
+
+* Build-essential
+* Qt5-qmake
+* Qtdeclarative5-dev
+* Qt5-default
+* Qttools5-dev-tools
+* Libjack-jackd2-dev
+
+#### On Fedora
+
+* Qt5-qtdeclarative-devel
+* Jack-audio-connection-kit-dbus
+* libQt5Concurrent5
+* Jack-audio-connection-kit-devel
+
+#### For all desktop distributions
+
+[QjackCtl](https://qjackctl.sourceforge.io/) is optional, but recommended to configure JACK.
+
+
+### Standard desktop build
+
+~~~
+qmake Jamulus.pro
+make clean
+make
+sudo make install
+~~~
+
+This puts the Jamulus binary into `/usr/local/bin`.
+
+#### Notes
+
+* The "make clean" command is essential to remove the automatically generated Qt files which are present in the .tar.gz file and may not match the Qt version you are using.
+
+* To use an external shared OPUS library instead of the built-in one use qmake `"CONFIG+=opus_shared_lib" Jamulus.pro`.
```

Comment:
Is it worth documenting this?

This may mostly be relevant for packagers which aim to avoid bundled libraries in most cases. Packers usually know the build systems and will hopefully spot this option.

For other users this is mainly confusing, I think. I used to be a packager for Gentoo, so my first reaction to seeing this option was "cool, I'm going to use it" until I ran into issues because the latest (?) publicly available libopus has a bug which makes using it from Jamulus impossible, IIRC.

So it might be best to remove this altogether or document it as "for packagers only" and reference the need for a very recent/patched libopus?

Reply:
I'm all in favour of removing stuff that's not helpful to the majority. If a packager wants to know they can always ask.

Reply:
Yep. I'd remove it then.